lighthouse-cli/cli-flags.js

```diff
@@ -12,19 +12,32 @@ const pkg = require('../package.json');
 const printer = require('./printer');
 
 /**
- * @param {string=} manualArgv
+ * @param {(string | string[])[]} arr
+ * @return {string[]}
+ */
+function flatten(arr) {
+  /** @type {string[]} */
+  const result = [];
+  return result.concat(...arr);
+}
+
+/**
+ * @param {string[]} manualArgv
  * @return {LH.CliFlags}
  */
-function getFlags(manualArgv) {
-  // @ts-ignore yargs() is incorrectly typed as not returning itself
-  const y = manualArgv ? yargs(manualArgv) : yargs;
-  return y.help('help')
+function getFlags(...manualArgv) {
+  const y = manualArgv.length ? yargs(manualArgv) : yargs;
```

Comment:
ha, actually, that test failure may be because our version of yargs doesn't take an array at all?

Reply:
It passes locally.

Reply:
ok the issue is that if you pass in an array, you cannot have each item have multiple args. test just needs to change

Reply:
also I guess they expect some preprocessing, b/c `=` is required too. https://github.com/GoogleChrome/lighthouse/pull/8933/commits/6f58b9007f2b9be45e7a399187b7d07b3d46554c

Reply:
> also I guess they expect some preprocessing, b/c = is required too.

You can only omit the `=` when passing in as a single string? This seems like the real downside...we should be able to write tests using both styles since the real deal accepts both